src/haxelib/SemVer.hx

```diff
@@ -97,6 +97,12 @@ abstract SemVer(String) to String {
 	inline function get_valid()
 		return isValid(this);
 
+	@:op(a > b) static function greaterThan(a:SemVer, b:SemVer) {
```

Comment:
I think this is wrong, since it doesn't check `preview`/`previewNum` fields. There's `Semver.compare(a,b)` that will return 1 if a > b.